app/components/common/ListItem.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { View, StyleSheet, TouchableOpacity, Dimensions } from 'react-native'
+import { scale } from 'react-native-size-matters'
+import RNText from './RNText'
+import colors from '../../config/colors'
+import Icon from 'react-native-vector-icons/Ionicons'
+
+const windowWidth = Dimensions.get('window').width
+const chevronIcon = (
```

Comment:
Why not sure have this inline?

Reply:
this is just from prettier config

Reply:
I mean why not just have this in the return JSX?